x-pack/platform/plugins/shared/cases/server/features/v3.ts

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import type { KibanaFeatureConfig } from '@kbn/features-plugin/common';
+import { hiddenTypes as filesSavedObjectTypes } from '@kbn/files-plugin/server/saved_objects';
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core/server';
+
+import { KibanaFeatureScope } from '@kbn/features-plugin/common';
+import { APP_ID, FEATURE_ID_V3 } from '../../common/constants';
+import { createUICapabilities, getApiTags } from '../../common';
+import {
+  CASES_DELETE_SUB_PRIVILEGE_ID,
+  CASES_SETTINGS_SUB_PRIVILEGE_ID,
+  CASES_CREATE_COMMENT_SUB_PRIVILEGE_ID,
+  CASES_REOPEN_SUB_PRIVILEGE_ID,
+  CASES_ASSIGN_SUB_PRIVILEGE_ID,
+} from './constants';
+
+/**
+ * The order of appearance in the feature privilege page
+ * under the management section. Cases should be under
+ * the Actions and Connectors feature
+ */
+
+const FEATURE_ORDER = 3100;
+
+export const getV3 = (): KibanaFeatureConfig => {
```

Comment:
Question: Are there tests to assert the differences between feature versions?

Reply:
not here, but in the api integration tests.